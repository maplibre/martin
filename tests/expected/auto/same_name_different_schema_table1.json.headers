content-encoding: br
content-type: application/json
etag: W/"180-QqfiHgGw9EHetkuLKz__dQ=="
transfer-encoding: chunked
vary: accept-encoding, Origin, Access-Control-Request-Method, Access-Control-Request-Headers
