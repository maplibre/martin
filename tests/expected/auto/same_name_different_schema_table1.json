{
  "bounds": [
    142.84094409430955,
    11.926741846036798,
    142.84143360001997,
    11.927370348564835
  ],
  "description": "schema_a.table_name_existing_two_schemas.geom",
  "name": "table_name_existing_two_schemas",
  "tilejson": "3.0.0",
  "tiles": [
    "http://localhost:3111/table_name_existing_two_schemas/{z}/{x}/{y}"
  ],
  "vector_layers": [
    {
      "fields": {
        "id": "int4",
        "name": "text"
      },
      "id": "table_name_existing_two_schemas"
    }
  ]
}
